### PR TITLE
feat: add font and desktop styling

### PR DESCRIPTION
**Pull request by believer**
Adds font Futura to globals and fixes some styling for desktop

### CODE REVIEW

File: lib/Header.tsx [hunk 1]
```diff
@@ -32,7 +32,6 @@ const Inner = styled.div`
 `
 
 const LogoWrap = styled.div`
-  background-color: #fff;
   box-shadow: 0 2px 10px rgba(0, 0, 0, 0.06);
```
Comment (AElmoznino) [inline lib/Header.tsx]: Should it really have the `box-shadow`? Looks kinda weird.
box-shadow

Comment (believer) [reply lib/Header.tsx]: Probably not, feel free to fix it!

Comment (AElmoznino) [reply lib/Header.tsx]: Cool, will do!

**Pull request merged**